README.rst

```diff
@@ -37,6 +37,32 @@ Create an Enum-class and pass it as first argument to the Django model EnumField
     class Beer(models.Model):
         style = enum.EnumField(BeerStyle, default=BeerStyle.LAGER)
 
+You can also set default value on your enum class using ``__default__``
+attribute
```

Comment:
Ok, so one could define a custom field for certain enum in case it's used in many places. Sure, let me discuss it with my colleagues to see if everyone is happy about it and redo the commits :smile: